### PR TITLE
VZ-11755: update guide to migrate OpenSearch on OCNE

### PR DESCRIPTION
**Pull request by karan56625**
PR includes:

- updated guide to migrate OpenSearch on OCNE under guide section.

### CODE REVIEW

File: content/en/docs/guides/migrate/install/opensearch/_index.md [hunk 1]
```diff
@@ -5,4 +5,952 @@ draft: false
...
+</div>  
+{{< /clipboard >}}
+{{< /alert >}}
+
+#### Create OpenSearch Cluster
+To create OpenSearch and OpenSearch Dashboards instance, you will require to create the OpenSearchCluster CR. 
+
+There are two ways to provide certificates in OpenSearchCluster CR.
+
+1. One is where you can allow the OpenSearch Operator to generate and sign the certificates. You can refer below CR that has self generated certificates.
```
Comment (pfmackin) [inline content/en/docs/guides/migrate/install/opensearch/_index.md:260]: Does this require cert-manager?  If so then this section needs to be moved to integration

Comment (karan56625) [reply content/en/docs/guides/migrate/install/opensearch/_index.md:260]: In the line 348, I have mentioned the link to the integration section as TBD. Once I add the integration section for OS, I will update that link.

File: content/en/docs/guides/migrate/install/opensearch/_index.md [hunk 2]
```diff
@@ -5,4 +5,952 @@ draft: false
...
+| nodePools[*].additionalConfig | additional configuration for the node group                                                | cluster.initial_master_nodes: opensearch-es-master-0                                                                                                                                              | ""                                                                                                                                                                                                                                                                                                                                                       | based on spec.profile                                                                                                                                                                                                               |
+
+
+##### OpenSearchCluster CR with Default values
+**Prod Profile**
+{{< clipboard >}}
+<div class="highlight">
+
+```
+apiVersion: opensearch.opster.io/v1
```
Comment (mgianatagh) [inline content/en/docs/guides/migrate/install/opensearch/_index.md:472]: can the extra lines between each entry be removed?

File: content/en/docs/guides/migrate/install/opensearch/_index.md [hunk 3]
```diff
@@ -5,4 +5,952 @@ draft: false
...
+</div>
+{{< /clipboard >}}
+
+**Dev Profile**
+{{< clipboard >}}
+<div class="highlight">
+
+```
+apiVersion: opensearch.opster.io/v1
+
```
Comment (mgianatagh) [inline content/en/docs/guides/migrate/install/opensearch/_index.md:645]: same comment as above about removing the blank lines

File: content/en/docs/guides/migrate/install/opensearch/_index.md (hunk 1 above)
Comment (pfmackin) [reply content/en/docs/guides/migrate/install/opensearch/_index.md:260]: If this has any external deps then it needs to be in the integration section.  You can move it once you work on that.

Comment (karan56625) [reply content/en/docs/guides/migrate/install/opensearch/_index.md:260]: ok sure.

File: content/en/docs/guides/migrate/install/opensearch/_index.md (hunk 2 above)
Comment (karan56625) [reply content/en/docs/guides/migrate/install/opensearch/_index.md:472]: fixed.

File: content/en/docs/guides/migrate/install/opensearch/_index.md (hunk 3 above)
Comment (karan56625) [reply content/en/docs/guides/migrate/install/opensearch/_index.md]: fixed

**Review by mgianatagh — APPROVED** [review]

**Pull request merged**